### ISSUE TITLE
cmip_fetch() with `overwrite = TRUE` is not re-fetching the data

### ISSUE THREAD

**Issue by cvitolo** [ropenscilabs/cmip#15]
This issue is part of [this rOpenSci review](https://github.com/ropensci/onboarding/issues/99).

I tried to run the following

`key <- "bcsd/yearly/cnrm_cm3.1/cnrm_cm3.1.sresa1b.monthly.Prcp.2034.nc"`
`res <- cmip_fetch(key)`
`res1 <- cmip_fetch(key, overwrite = TRUE)`

But noticed that the second time, it is not re-fetching the data. Is this the expected behaviour?

Maybe mention in the documentation that the data are going to be cached?

**Comment by bpbond** [comment]
Yes. I also couldn't get the `"file exists and ovewrite != TRUE"` message to print, so probably look hard at this logic/documentation.

**Comment by bpbond** [comment]
Note also there's no test that checks whether `overwrite` works or not.
